site/content/in-dev/unreleased/getting-started/_index.md

```diff
@@ -23,3 +23,114 @@ weight: 101
 build:
   render: never
 ---
+
+# Getting Started with Apache Polaris Binary Distribution
+
+Quickly start Apache Polaris by running the pre-built binary, no build needed.
+
+---
+
+## Prerequisites
+
+- Java 21 or later installed. You can verify this by running:
+
+```bash
+java -version
+```
+
+- Basic familiarity with running commands in your operating system terminal.
+
+---
+
+## Step 1: Download the Apache Polaris Binary
+
+1. Visit the official Apache Polaris GitHub
+   [Releases page](https://github.com/apache/polaris/releases).
+
+2. Download the latest binary archive file, for example:
+
+```bash
+curl -L https://downloads.apache.org/incubator/polaris/1.0.0-incubating/polaris-bin-1.0.0-incubating.tgz | tar xz```
+---
+
+## Step 2: Extract the Archive
+
+Extract the downloaded tar.gz file to your desired directory:
```

Comment:
We actually combine two steps as a single command handles both download and exacting. Let's remove this step. The `cd` part can go to next step.